---
  # These constants are in use in PandaPose.py File, to set global sleep time and frequency
  zero_g_poses: 20
  dynamic_frequency: 100.0
  gravity_constant: 9.81
  oscillation_magnitude:
    - 1.5
    - 1.5
    - 1.5
    - 1.5
    - 1.5
    - 1.5
    - 1.5

  oscillation_frequency:
    - 1.0
    - 1.0
    - 1.0
    - 1.0
    - 1.0
    - 1.0
    - 1.0

  oscillation_time: 3.0

  is_sim: true
  rest_time: 2
